uint8 id_mcpf
uint8 id_mcpa
uint8 id_pip
uint8 id_dip
int16 pos_mcpf
int16 pos_mcpa
int16 pos_pip
int16 pos_dip